functional/PutObjectRunnable.java

```diff
@@ -37,8 +38,10 @@ public void run() {
     StringBuffer traceBuffer = new StringBuffer();
 
     try {
+      Path filePath = Paths.get(filename);
+      RandomAccessFile file = new RandomAccessFile(filePath.toFile(), "r");
```

Comment:
Why is it needed? can't we support filename upload version of put object?

Reply:
All the validations moved to
 ``` public void putObject(String bucketName, String objectName, Object data, Long size,
                        Map<String, String> headerMap, ServerSideEncryption sse, String contentType) ```